src/AppBundle/Controller/SecurityController.php

```diff
@@ -13,6 +13,7 @@
 
```

Comment:
Yes this is to show that we can easily inject dependencies into controllers. In this example, the `AuthenticationUtils` service is always used.